Quaver.Shared/Screens/Settings/SettingsDialog.cs

```diff
@@ -334,7 +334,8 @@ private void CreateSections()
                     new SettingsBool(this, "Top Lane Cover", ConfigManager.LaneCoverTop),
                     new SettingsBool(this, "Bottom Lane Cover", ConfigManager.LaneCoverBottom),
                     new SettingsBool(this, "Display UI Elements Over Lane Covers", ConfigManager.UIElementsOverLaneCover),
-                    new SettingsBool(this, "Smooth Accuracy Changes", ConfigManager.SmoothAccuracyChanges)
+                    new SettingsBool(this, "Smooth Accuracy Changes", ConfigManager.SmoothAccuracyChanges),
+                    new SettingsBool(this, "Show Hit Accuracy", ConfigManager.ShowHitAccuracy)
```

Comment:
A SettingsBool is probably not the best way to do it since it would be better to toggle individual judgements, but this would require having some sort of checkbox dropdown or field with multiple checkboxes which isn't implemented yet. It works for now however if we're only trying to hide marvs & perfs.